### ISSUE TITLE
Dictionary doesn't show more than 10 entries for words with multiple meanings

### ISSUE THREAD

**Issue by DiamondNg** [xiaoyifang/goldendict-ng#2026]
**Describe the bug**
For words which have more than 10 meanings, each on a separate entry, Golden Dictionary doesn't show more than 10 entries.

**To Reproduce**
1. Go search "dhamma" 
2. Scroll down to see only 10 entries for that word. 
3. Whereas the word dhamma has 15 meanings in Digital Pali Dictionary, search "dhamma 1.15" you will see an entry.

**Affected Dictionaries**

Digital Pali Dictionary (https://digitalpalidictionary.github.io/) download available in the link.

If possible, please attach the dictionary files or provide a downloadable link. (GitHub allows uploading 25MiB zip file as attachment)

**Expected behavior**
I expect to see all the meanings available in the dictionary to be displayed when I search for "dhamma", from dhamma 1.01 to dhamma 1.15.

**Screenshots**

**OS and software versions**

- OS: Windows10 pro
- Version 22H2

**Additional context**

Platform | Context
---------|----------
 Windows |Upload the dmp file in the crash folder located in the configuration folder (can be opened through the menu help->configuration folder) to here
 Macos | upload the crashpad crash log
 Linux | gdb backtrace info

**Comment by xiaoyifang** [comment]
![image](https://github.com/user-attachments/assets/a4f7a012-50d9-4c1a-b794-3b1255e28b74)

not reproduced.

**Comment by Devamitta** [comment]
same issue on the Linux mint 22.1 Cinnamon

Goldendict-ng version: 25.02.0.214573d1 at 2025-01-24T19:22:50Z

<img width="368" alt="Image" src="https://github.com/user-attachments/assets/00686c05-7fc5-45d7-b349-5fa8c6dbe636" />

**Comment by xiaoyifang** [comment]
seems not the same dictionary ,  show me the dictionary link.

**Comment by Devamitta** [comment]
I installed if from flatpak:

flatpak install io.github.xiaoyifang.goldendict_ng

**Comment by xiaoyifang** [comment]
I mean the dictionary may be has some issue.

**Comment by bdhrs** [comment]
@xiaoyifang I have the same issue, only ten items displaying, with the same word as above. I see this in the terminal when I run `goldendict "dhamma"`
```bash
js: ResizeObserver loop limit exceeded
article view loaded url: "gdlookup://localhost/?word=dhamma&group=4294967294" true
```

surely `js: ResizeObserver loop limit exceeded` is causing the limit? How to change that?

**Comment by xiaoyifang** [comment]
I do not think this is the cause ,Change the dictionary. or use other software to verify the output result .
